app/assets/stylesheets/welcome.scss

```diff
@@ -36,3 +37,21 @@ input#original_url {
 .urls-index-header {
   margin-left: 700px;
 }
+
+input#url_original_url {
```

Comment:
Avoid using id selectors<br>Avoid qualifying id selectors with an element.<br>Selector `url_original_url` should be written in lowercase with hyphens